packages/arb-bridge-peripherals/contracts/tokenbridge/libraries/TransferAndCallToken.sol

```diff
@@ -2,11 +2,14 @@
 pragma solidity >0.6.0 <0.8.0;
 
 import "@openzeppelin/contracts-upgradeable/token/ERC20/ERC20Upgradeable.sol";
-import "./IERC677.sol";
+import "./ITransferAndCall.sol";
 
-// Implementation from https://github.com/smartcontractkit/LinkToken/blob/master/contracts/v0.6/ERC677Token.sol
-
-abstract contract ERC677Token is ERC20Upgradeable, IERC677 {
+// Implementation from https://github.com/smartcontractkit/LinkToken/blob/master/contracts/v0.6/TransferAndCallToken.sol
+/**
+ * @notice based on Implementation from https://github.com/smartcontractkit/LinkToken/blob/master/contracts/v0.6/ERC677Token.sol
+ * Implementation doesn't return a bool on onTokenTransfer, and so doesn't adhere to the proposed 677 standard, thus, tho it's similar to ERC677, we don't refer to it as such.
```

Comment:
This sentence is a bit long and confusing.
I would say something like
" The implementation doesn't return a bool on onTokenTransfer. This is similar to the proposed 677 standard, but still incompatible - thus we don't refer to it as such."